flaxandteal/coral-arches
# Fix/#2362/disable allow instance creation

Uses nodeOptions to disable instance creation across the site.

To test simply kill containers
`make webpack`
`make manage CMD="coral reload"`

Then check each of the dropdowns listed in [#2362](https://tree.taiga.io/project/viktoriabon-coral-phase-2/issue/2362)
